modules/microshift-mirroring-prereqs.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: CONCEPT
+[id="microshift-mirroring-prereqs_{context}"]
+= Mirroring prerequisites
+
+Follow the instructions in the "Configuring credentials that allow images to be mirrored" link provided in the Additional resources to create a `~/.pull-secret-mirror.json` file including the user credentials for accessing the mirror.
+
+For example, the following section is added to the pull secret file for the `microshift-quay:8443` mirror registry using `microshift:microshift` as username and password.
+
+.Example mirror registry section for pull secret file
+
+[source,terminal]
+----
+"<microshift-quay:8443>": { <1>
+    "auth": "<bWljcm9zaGlmdDptaWNyb3NoaWZ0>",
+    "email": "<microshift-quay@example.com>"
+},
+----
+<1> Replace <microshift-quay:8443> with the host name and port of your mirror registry server.
```

Comment:
Everything should be replaced in this file: auth and email values included.
How can we adjust this section to reflect this better?